src/components/landing/Footer.js

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
+import H5 from '@material-tailwind/react/Heading5';
+import LeadText from '@material-tailwind/react/LeadText';
+import Icon from '@material-tailwind/react/Icon';
+
+export default function Footer() {
+    return (
+        <>
+            <footer className="relative bg-gray-50 pt-8 pb-6">
+                <div className="container max-w-7xl mx-auto px-4">
+                    <div className="flex flex-wrap text-center lg:text-left pt-6">
+                        <div className="w-full lg:w-6/12 px-4">
+                            <H5 color="gray">Material Tailwind</H5>
+                            <LeadText
+                                color="blueGray"
+                                style={{ marginTop: '0' }}
```

Comment:
Hey dude. When I started to work with React, there were a lot of people that were not happy with using `style` (inline style). I think you could solve this by wrapping the `LeadText` component inside a `div` component with the class of `-mt-*`, the negative margin-top class of the normal one that `LeadText` uses.
Makes sense?